tests/schemas/number/PositiveInt.test.ts

```diff
@@ -1,50 +1,52 @@
 import { unsafeCoerce } from 'fp-ts/function'
-import * as PositiveInt from '../../src/number/positiveInt'
-import { validateArbitrary } from '../../test-utils'
+import { PositiveInt } from '../../../src/schemas/number/PositiveInt'
+import { getAllInstances, validateArbitrary } from '../../../test-utils'
 
-const _: (n: number) => PositiveInt.PositiveInt = unsafeCoerce
+const _: (n: number) => PositiveInt = unsafeCoerce
+
+const { Decoder, Eq, Guard, Arbitrary, Type } = getAllInstances(PositiveInt)
 
 describe('PositiveInt', () => {
   describe('Decoder', () => {
     it('catches an invalid PositiveInt', () => {
-      const result = PositiveInt.Decoder.decode(-1)
+      const result = Decoder.decode(-1)
       expect(result._tag).toBe('Left')
     })
     it('validates a valid PositiveInt', () => {
-      const result = PositiveInt.Decoder.decode(1)
+      const result = Decoder.decode(1)
       expect(result._tag).toBe('Right')
     })
   })
   describe('Eq', () => {
     it('returns true for similar PositiveInts', () => {
-      expect(PositiveInt.Eq.equals(_(1), _(1))).toBe(true)
+      expect(Eq.equals(_(1), _(1))).toBe(true)
     })
     it('returns false for dissimilar PositiveInts', () => {
-      expect(PositiveInt.Eq.equals(_(1), _(2))).toBe(false)
+      expect(Eq.equals(_(1), _(2))).toBe(false)
     })
   })
   describe('Guard', () => {
     it('guards against invalid PositiveInt', () => {
-      expect(PositiveInt.Guard.is(1.1)).toBe(false)
+      expect(Guard.is(1.1)).toBe(false)
     })
     it('permits a valid PositiveInt', () => {
-      expect(PositiveInt.Guard.is(1)).toBe(true)
+      expect(Guard.is(1)).toBe(true)
     })
   })
   describe('Type', () => {
     it('decodes an invalid PositiveInt', () => {
-      const result = PositiveInt.Type.decode(-4)
+      const result = Type.decode(-4)
       expect(result._tag).toBe('Left')
     })
     it('decodes an valid PositiveInt', () => {
-      const result = PositiveInt.Type.decode(4)
+      const result = Type.decode(4)
       expect(result._tag).toBe('Right')
     })
   })
 
   describe('Arbitrary', () => {
     it('generates valid PositiveInts', () => {
-      validateArbitrary(PositiveInt, PositiveInt.isPositiveInt)
+      validateArbitrary({ Arbitrary }, Guard.is)
```

Comment:
the reason I made `validateArbitrary` take an `{ Arbitrary }` shaped object was to be able to import an entire instance into some namespace and just pass in the namespace. also works for the return of `getAllInstances`

if you're not going to do that (which is fine) you might want to tweak `validateArbitrary` so it doesn't look so awkward to call